.github/workflows/labeler.yml

```diff
@@ -7,6 +7,10 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        persist-credentials: false
```

Comment:
Is it also worth specifying `sparse-checkout: . github/labeler.yml` and `sparse-checkout-cone-mode: false`? xref https://github.com/actions/checkout?tab=readme-ov-file#fetch-only-a-single-file

Reply:
I suppose so; I was just parroting the documentation 😅